environments/softlayer/inventory.ini

```diff
@@ -15,7 +15,7 @@
 
 [db0:vars]
 datadog_integration_cloudant=true
-encrypted_root=/opt/data
```

Comment:
Can you explain why this wasn't working? Does it have to do with the subtle `kafka_log_dir` change below?

Reply:
It was just a variable precedence issue. Variables loaded from the vars files override anything that's in the inventory so the options were:

1. remove the var from `public.yml` and only specify it in the inventory file
2. change the var name and check if it's already defined in the vars file 

Option 2 seemed like the better option.

For reference: https://docs.ansible.com/ansible/latest/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable